### ISSUE TITLE
Implement details of MMN w/o BS

### ISSUE THREAD

**Issue by nbgao** [wenhuchen/Meta-Module-Network#9]
![image](https://user-images.githubusercontent.com/15265319/169821165-5ea0c8ed-0378-4722-a320-fc8574b59041.png)
The GQA testdev accuracy of MMN without Bootstrap is 58.4% in your paper, however, I only reached 57.7% with your provided vision features under the default setup.
My primary hyperparameters as follows:
  - model=TreeSparsePostv2
  - pre_layers=3
  - stacking=2
  - lr_default=1e-4
  - batch_size=256

Could you please tell me more details about this experiment result?

**Comment by nbgao** [comment]
Besides, the best testdev accuracy of your provided pre-trained models **TreeSparsePost2FullValidationRemoved** is **model_ep4_0.5623**, why this result is not matching to **56.5** of _MMN w/o FT_ ?

**Comment by wenhuchen** [comment]
The models released in the github is trained on training set only. The paper uses the model trained on the combination of training+validation set, which bumps the score by 2%. "TreeSparsePost2FullValidationRemoved" means the full validation set is removed during training.

**Comment by dreichCSL** [comment]
Are the programs that are shared in this repo also generated by a model that was trained only on train? So, all json files with "fully" in their name (e.g. trainval_unbiased_fully_inputs.json) are generated by a model that was **not** trained on val set?
